__tests__/business-text/business-text.variables.js

```diff
@@ -0,0 +1,55 @@
+const newBusinessText = {
+  "title": [
```

Comment:
Please, get rid of quotes for object keys, and for values leave single quotes where it is needed